Code/training_team8.py

```diff
@@ -24,8 +25,8 @@
 # %% ----------------------------------- Hyper Parameters --------------------------------------------------------------
 LR = 5e-3
 MOMENT = .9
-BATCH_SIZE = 8
-N_EPOCHS = 50
+BATCH_SIZE = 16
```

Comment:
increased this because my hardware supports it, and it'll speed things up. can be lowered if we run into CUDA ram errors.